KPI-team1-frontend/src/components/Sidebar.tsx

```diff
@@ -56,7 +59,7 @@ export default function Sidebar({
             </NavLink>
             <div>
               <NavLink
-                to={"/kpi/1"}
+                to={"/kpi/circles"}
```

Comment:
i'm guessing that we still want to have a default "view" if we click on KPIs button 
right now it to shows the uncomplete page 
It this is the goal, we should add a todo to make sure we have something on that page

Reply:
yes, we still have a default "view" when click on KPIs button and it was been done by this block of code in KpiPage:
`export default function KpiPage(): JSX.Element {
  const { circleId } = useParams();
  const [selectedCircleId, setSelectedCircleId] = useState<string | null>("");
  const [modalIsOpen, setModalIsOpen] = useState(false);
  const [selectedKpi, setSelectedKpi] = useState<KpiExtended | null>(null);
  const { kpiDefinitions, userDetails }: OutletContext =
    useOutletContext();

  useEffect(() => {
    if (circleId) {
      setSelectedCircleId(circleId);
    } else {
      setSelectedCircleId(userDetails.defaultCircleId);
    }
  }, [circleId]);`